lib/bigcommerce/prometheus/configuration.rb

```diff
@@ -42,6 +42,7 @@ module Configuration
         server_port: ENV.fetch('PROMETHEUS_SERVER_PORT', PrometheusExporter::DEFAULT_PORT).to_i,
         server_timeout: ENV.fetch('PROMETHEUS_DEFAULT_TIMEOUT', PrometheusExporter::DEFAULT_TIMEOUT).to_i,
         server_prefix: ENV.fetch('PROMETHEUS_DEFAULT_PREFIX', PrometheusExporter::DEFAULT_PREFIX).to_s,
+        server_thread_pool_size: ENV.fetch('PROMETHEUS_SERVER_THREAD_POOL_SIZE', 20).to_i,
```

Comment:
We can probably drop the default down to like 3 or so. 20 seems excessive for a single endpoint hit once

Reply:
Im open to this -- just left at 20 because that is the current default value. Do you think just a note in the changelog is sufficient to let people know that the default is changing?

Reply:
Going to also update readme with some info on this new config

Reply:
> Im open to this -- just left at 20 because that is the current default value. Do you think just a note in the changelog is sufficient to let people know that the default is changing?

Yup, but we should bump the minor version since it's a behavioral change.